climada/util/coordinates.py

```diff
@@ -2646,36 +2646,21 @@ def set_df_geometry_points(df_val, scheduler=None, crs=None):
     ----------
     df_val : GeoDataFrame
         contains latitude and longitude columns
-    scheduler : str, optional
-        used for dask map_partitions. “threads”, “synchronous” or “processes”
+    scheduler : str
+        deprecated, obsolete
```

Comment:
Use Sphinx directive
```suggestion
    scheduler : str, optional
        Scheduler type for dask map_partitions.

        .. deprecated:: 4.1
           This function does not use dask features anymore. The parameter has no effect
           and will be removed in a future version.
```